databases/week1/hw-week1.sql

```diff
@@ -0,0 +1,53 @@
+-- 1 Find out how many tasks are in the task table
+SELECT COUNT(id) 
+FROM `task`;
+
+-- 2 Find out how many tasks in the task table do not have a valid due date
+SELECT COUNT(id) 
+FROM `task` 
+WHERE `due_date` is NULL;
+
+-- 3 Find all the tasks that are marked as done
+SELECT task.title, task.description, status.name
+FROM `task`
+JOIN `status` ON task.status_id = status.id
+WHERE `name` REGEXP 'done';
+
+-- 4 Find all the tasks that are not marked as done
+SELECT task.title, task.description, status.name
+FROM `task`
+JOIN `status` ON task.status_id = status.id
+WHERE `name` NOT REGEXP 'done';
+
+-- 5 Get all the tasks, sorted with the most recently created first
+SELECT *
+FROM `task`
+ORDER BY `created` DESC;
+
+-- 6 Get the single most recently created task
+SELECT MAX(created) 
+FROM `task`;
```

Comment:
Here no one knows which one of the tasks is that(at least task id needed)
Try this if it makes sense
```
SELECT id,title,created
FROM task
ORDER BY created DESC LIMIT 1;
```

Reply:
Thank you for your review. I agree with you, it looks better.